dialect/sql/schema/postgres.go

```diff
@@ -241,7 +241,7 @@ func (d *Postgres) scanColumn(c *Column, rows *sql.Rows) error {
 		c.Size = maxCharSize + 1
 	case "character", "character varying":
 		c.Type = field.TypeString
-	case "date", "time", "timestamp", "timestamp with time zone":
+	case "date", "time", "timestamp", "timestamp with time zone", "timestamp without time zone":
```

Comment:
should timestamptz variations also be included?

Reply:
I'm not sure. We need to check what is returned from `INFORMATION_SCHEMA` if the column was created with the `timestamptz` abbreviation.

Reply:
@ssoor, you're welcome to check that and contribute this in a different PR if it's needed.